<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>787</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Receive Window</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="ReceiveWin">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>400</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Send Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Send_S_Button">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Stop_S_Button">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="SendWindow"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_N">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="action_Open_file"/>
    <addaction name="action_Save_file"/>
    <addaction name="separator"/>
    <addaction name="actionclose"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionReceive_R"/>
    <addaction name="actionPause_P"/>
    <addaction name="actionStop_S"/>
    <addaction name="separator"/>
    <addaction name="actionClear_C"/>
    <addaction name="separator"/>
    <addaction name="actionFind_F"/>
    <addaction name="separator"/>
    <addaction name="actionSetting_S"/>
    <addaction name="actionOpen_Dock"/>
    <addaction name="actionClose_Dock"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh_R"/>
   </widget>
   <widget class="QMenu" name="menuSetting_S">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionDefault_Setting"/>
   </widget>
   <widget class="QMenu" name="menuAbout_A">
    <property name="title">
     <string>About(&amp;A)</string>
    </property>
    <addaction name="actionTouch_us"/>
   </widget>
   <addaction name="menu_N"/>
   <addaction name="menuEdit_E"/>
   <addaction name="menuSetting_S"/>
   <addaction name="menuAbout_A"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh_R"/>
   <addaction name="actionStart"/>
   <addaction name="separator"/>
   <addaction name="actionReceive_R"/>
   <addaction name="actionPause_P"/>
   <addaction name="actionStop_S"/>
   <addaction name="separator"/>
   <addaction name="actionClear_C"/>
   <addaction name="actionSetting_S"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>612</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Port Setting</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="verticalSpacing">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>PortNum</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="PortNumBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Buad rate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="BuatrateBox">
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>DataBits</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="DataBitBox">
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>StopBit</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="StopBitBox">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>CheckBit</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="CheckBitBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Space</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>FlowCtrl</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="FlowCtrl">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RTS/CTS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XON/XOFF</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Receive Setting</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QRadioButton" name="R_ASSIC">
          <property name="text">
           <string>ASSIC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="R_HEX">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="AutoLineFeed">
          <property name="text">
           <string>AutoLineFeed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ShowTimeCheck">
          <property name="text">
           <string>ShowTime</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Send Setting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="3">
         <widget class="QCheckBox" name="TimeCheck">
          <property name="text">
           <string>AutoSend</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLineEdit" name="TimeEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="S_ASSIC">
          <property name="text">
           <string>ASSIC</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="S_HEX">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Send Histiry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="HistoryWin"/>
       </item>
       <item>
        <widget class="QPushButton" name="ClearHistory">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,1">
       <item>
        <widget class="QLineEdit" name="CountEdit">
         <property name="text">
          <string>input data</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ShowDataNumLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CountCheck">
         <property name="text">
          <string>Count(must open AutoLineFeed)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearCountButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open_file">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/new_file.png</normaloff>:/image/images/new_file.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="action_Save_file">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/close.png</normaloff>:/image/images/close.png</iconset>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/start.png</normaloff>:/image/images/start.png</iconset>
   </property>
   <property name="text">
    <string>Start(&amp;S)</string>
   </property>
  </action>
  <action name="actionPause_P">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/pause.png</normaloff>:/image/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause(&amp;P)</string>
   </property>
  </action>
  <action name="actionStop_S">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/stop.png</normaloff>:/image/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop(&amp;S)</string>
   </property>
  </action>
  <action name="actionClear_C">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/clean.png</normaloff>:/image/images/clean.png</iconset>
   </property>
   <property name="text">
    <string>Clear(&amp;C)</string>
   </property>
  </action>
  <action name="actionFind_F_2">
   <property name="text">
    <string>Find(&amp;F)</string>
   </property>
  </action>
  <action name="actionFind_F">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/find.png</normaloff>:/image/images/find.png</iconset>
   </property>
   <property name="text">
    <string>Find(&amp;F)</string>
   </property>
  </action>
  <action name="actionSetting_S">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/set.png</normaloff>:/image/images/set.png</iconset>
   </property>
   <property name="text">
    <string>Setting(S)</string>
   </property>
  </action>
  <action name="actionTouch_us">
   <property name="text">
    <string>Contact us</string>
   </property>
  </action>
  <action name="actionDeclare">
   <property name="text">
    <string>Declare</string>
   </property>
  </action>
  <action name="actionDefault_Setting">
   <property name="text">
    <string>Default Setting</string>
   </property>
  </action>
  <action name="actionOpen_Dock">
   <property name="text">
    <string>Open Dock</string>
   </property>
  </action>
  <action name="actionClose_Dock">
   <property name="text">
    <string>Close Dock</string>
   </property>
  </action>
  <action name="actionRefresh_R">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/shuxin.png</normaloff>:/image/images/shuxin.png</iconset>
   </property>
   <property name="text">
    <string>Refresh(&amp;R)</string>
   </property>
  </action>
  <action name="actionReceive_R">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/receive.png</normaloff>:/image/images/receive.png</iconset>
   </property>
   <property name="text">
    <string>Receive(&amp;R)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
